src/popup/app.tsx

```diff
@@ -24,8 +24,9 @@ export function AppContents({ popupWindow, storeState, dispatchUserActions }: Ap
 }
 
 export function App(props: AppProps): JSX.Element {
+  const theme = props.storeState.darkModeOn ? 'dark-theme' : ''
   return (
-    <div className="app">
+    <div className={`app ${theme}`}>
```

Comment:
we have 2 `div`s with the `app` class name. Is this intentional?

Reply:
Are you referring to the one in popup.html? That one will get replaced by this one once the `ReactDOM.render` runs